pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -295,27 +295,46 @@ def _check_config_and_set_final_flags(
             self._precision_flag = precision
 
         if plugins:
+            plugins_flags_count = {
+                "strategy": 0,
+                "precision_plugin": 0,
+                "precision": 0,
+                "checkpoint_io": 0,
+                "cluster_env": 0,
```

Comment:
the keys here should be the base interface name. for instance, users know ClusterEnvironment, not "cluster_env" . this will improve the error message below